cookbooks/bcpc-hadoop/templates/default/hdp_ldap-conn-pass.txt.erb

```diff
@@ -0,0 +1 @@
+<%= node[:bcpc][:hadoop][:hdfs][:ldap][:password] %>
```

Comment:
Do we need a template or could this be a file resource with a contents attribute?